src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py

```diff
@@ -724,12 +728,16 @@ def mocked_log_error(*args, **kwargs):  # pylint: disable=unused-argument
     self.mock.log_error.side_effect = mocked_log_error
     _, corpus_path = setup_testcase_and_corpus('empty',
                                                'corpus_with_some_files')
-    os.environ['EXIT_FUZZER_CODE'] = exit_code
```

Comment:
This was wrong, as it wont work inside minijail, so passed it properly with options.extra_env